libraries/AP_ESC_Telem/AP_ESC_Telem.h

```diff
@@ -9,6 +9,10 @@
 
 #ifdef NUM_SERVO_CHANNELS
  #define ESC_TELEM_MAX_ESCS NUM_SERVO_CHANNELS
+#elif defined(HAL_BUILD_AP_PERIPH) && defined(HAL_PWM_COUNT)
```

Comment:
better to add a SRV_Channel/SRV_Channel_config.h

Reply:
Added `SRV_Channel/SRV_Channel_config.h` and included where applicable.